packages/stepper/src/components/StepperForm/StepFooter/StepFooter.component.tsx

```diff
@@ -0,0 +1,49 @@
+import { ButtonPrimary, ButtonSecondary, Divider, StackHorizontal } from '@talend/design-system';
+import { useContext } from 'react';
+import { useTranslation } from 'react-i18next';
+import I18N from '../../../constants/i18n';
+import { StepperFormContext } from '../StepperForm.context';
+import style from '../StepperForm.module.scss';
+import { StepFooterProps } from './StepFooter.types';
```

Comment:
What do you think adding 
`"@trivago/prettier-plugin-sort-imports": "^3.4.0",` in the package.json in devdep ?

Doing it so, you'll have the import this way :

```
import { useContext } from 'react';
import { useTranslation } from 'react-i18next';

import { ButtonPrimary, ButtonSecondary, Divider, StackHorizontal } from '@talend/design-system';

import I18N from '../../../constants/i18n';
import { StepperFormContext } from '../StepperForm.context';
import { StepFooterProps } from './StepFooter.types';

import style from '../StepperForm.module.scss';
```